sklearn/feature_selection/_rfe.py

```diff
@@ -624,6 +628,15 @@ class RFECV(RFE):
     array([1, 1, 1, 1, 1, 6, 4, 3, 2, 5])
     """
 
+    _parameter_constraints = {
+        **RFE._parameter_constraints,  # type: ignore
```

Comment:
For me, `mypy` passes without the `# type: ignore` comment. Do you see a mypy error without the comment?

Reply:
I'll merge this first, because using  `# type: ignore` is consistent with what other estimators are doing.

I opened https://github.com/scikit-learn/scikit-learn/pull/24301 to address this typing issue everywhere.